cpp/src/DataStorm/SessionI.cpp

```diff
@@ -216,7 +218,7 @@ SessionI::attachTags(int64_t topicId, ElementInfoSeq tags, bool initialize, cons
 
     runWithTopics(
```

Comment:
We have two overloads of `runWithTopics` which are almost identical, only differing in the passed callback, one has an extra `TopicSubscribers` parameter that was never used. I got read of that overload and updated the callbacks to use the other one.